extension/llm/runner/stats.h

```diff
@@ -75,6 +83,11 @@ struct ET_EXPERIMENTAL Stats {
     aggregate_sampling_time_ms = 0;
     num_prompt_tokens = 0;
     num_generated_tokens = 0;
+    gpu_total_bytes = static_cast<uint64_t>(-1);
+    gpu_free_before_load_bytes = static_cast<uint64_t>(-1);
+    gpu_free_after_load_bytes = static_cast<uint64_t>(-1);
+    gpu_free_after_generate_bytes = static_cast<uint64_t>(-1);
+    gpu_peak_usage_mb = -1.0;
```

Comment:
I think we can rename it for better description. I don't think it is actually the peak gpu usage, it is the memory usage gap between start and finish running. It could be happen if the peak moment happen in the middle of execution. 

Same as other places for "peak gpu memory"